data/recipes/gcp_logging_cloudaudit_ts.json

```diff
@@ -14,7 +14,9 @@
         "name": "GCPLogsCollector",
         "args": {
             "project_name": "@project_name",
-            "filter_expression": "logName=projects/@project_name/logs/cloudaudit.googleapis.com%2Factivity timestamp>\"@start_date\" timestamp<\"@end_date\""
+            "filter_expression": "@filter_expression",
```

Comment:
Here and in other recipes: Any arguments to be interpolated (that is, `@arg_name`) into a recipe need to be added into the `"args"` section. In this recipe, that would be `filter_expression`, `backoff` and `delay`.

Also, do we want to change up the filter expression here anyway? I didn't author the relevant module, so I'm not sure why the original `filter_expression` takes the form it does, but I assume there must have been a reason. How does changing this affect log collection? I notice `data/recipes/gcp_logging_collect.json` has the form you are changing to, so I assume it's probably fine, but we should make sure we're not breaking any expected behaviour.